addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2739,6 +2739,194 @@ options.registry.ReplaceImage = options.Class.extend({
     },
 });
 
+options.registry.ConditionalVisibility = options.Class.extend({
+
+    /**
+     * We clear out the current style tag to make everything visible
+     * @override
+     */
+    init: function () {
```

Comment:
You can use ES6 syntax for all your method definitions:
```javascript
init() {
    // ...
},
```
(no function keyword)